build.sh

```diff
@@ -3,7 +3,7 @@
 rm -rf node_modules
 
 TIME="$(date +%s)"
-yarn --production
+npm install --production
```

Comment:
```suggestion
npm install --only=production
```

Reply:
According to the documentation, the `--production` is ok